_posts/en/newsletters/2024-12-06-newsletter.md

```diff
@@ -0,0 +1,117 @@
+  the same difficulty period _N_".
+
+  Anthony Towns [favored][towns time warp] an alternative solution
+  proposed by Zawy that would forbid any block from claiming it was
+  produced more than two hours before any previous block.  Zawy
+  [noted][zawy time warp] that his every-block solution increased the
+  risk of miners losing money from running outdated software but it
+  would also make timestamps more accurate for other uses, such as time
+  locks (e.g., [HTLCs][topic htlc]).
+
+  Separately, both on [Delving Bitcoin][poinsot duptx delv] and the
+  [Bitcoin-Dev mailing list][poinsot duptx ml], Poinsot sought feedback
+  about which proposed solution to use to prevent block 1,983,702 and
+  some later blocks from [duplicating][topic duplicate transactions] a
+  previous coinbase transaction (which could lead to loss of money and
+  the creation of an attack vector).  Four proposed solutions are
+  presented, all of which would only directly affect miners---so
+  feedback from miners would be especially appreciated.
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [Core Lightning 24.11rc2][] is a release candidate for the next major
+  version of this popular LN implementation.
+
+- [LDK 0.18.4-beta.rc1][] is a release candidate for a minor version of
+  this popular LN implementation.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #30708][] adds the `getdescriptoractivity` RPC command that
+  retrieves all transactions associated with a [descriptor][topic descriptors]
+  within a specified set of blockhashes, allowing wallets to interact with
+  Bitcoin Core in a stateless manner. This command is particularly useful when
+  used in conjunction with `scanblocks` (See Newsletter [#222][news222
+  scanblocks]), which identifies the set of blockhashes containing transactions
+  associated with a descriptor.
+
+- [Core Lightning #7832][] introduces [anchor outputs][topic anchor outputs] for
+  non-urgent unilateral close transactions, starting with a block target of 2016
+  blocks (approximately 2 weeks) and gradually reducing to 12 blocks. The
+  broadcast timestamp will be tracked to ensure consistent behaviour across
+  restarts. Previously, these transactions did not have anchor outputs by
```

Comment:
I think this is incorrect.  Anchor-style channels must have anchor _outputs_ and, since unilateral closes are signed by both parties, no single party can decide not to include them.  What I think this PR does is always create a spend of the local party's anchor output (a child transaction).  That way the user can either RBF the child for higher CPFP fee bumping or the user can CPFP the child (if the carve out limits haven't been hit).

I'm going to make edits to the above effect, but I wanted to leave this note in case I'm wrong.